oam/install_scripts/post-install

```diff
@@ -123,7 +123,7 @@ cd /
 
 test -d /var/log/mariadb || $SUDO mkdir /var/log/mariadb >/dev/null 2>&1
 test -d /var/log/mariadb/columnstore || $SUDO mkdir /var/log/mariadb/columnstore >/dev/null 2>&1
-$SUDO chmod -R 777 /var/log/mariadb
+$SUDO chmod -R 755 /var/log/mariadb
```

Comment:
777 should be fine for this. Unless we chown /var/log/mariadb somewhere I can't see.